cherryontech/jupiter-jumpers
# Loading

## Description

1. Adds a loading component at login, to allow the calendar to completely render before the user can create a log.

## Related Issue

N/A

## Related Story Card

[SCRUM 58](https://cherryontech-jupiter-jumpers.atlassian.net/browse/SCRUM-58)

## Type of Changes

_Feature update/change/addition, bug fix, styling update, etc._

## Acceptance Criteria

- [X]     Given I have just logged in
- [X]     When I am redirected to my dashboard
- [X]     Then I can see a brief “Loading” component (screen/pop-up), which is shown until the website is completely loaded, and then disappears.

## Testing Instructions

1. Log into the website.
2. Ensure you see a loading screen
3. Ensure the loading screen goes away, and that you can see a fully-loaded calendar.
